### ISSUE TITLE
Is JSHint really the "best" linter?

### ISSUE THREAD

**Issue by NateEag** [braziljs/js-the-right-way#205]
As far as I'm aware, [ESLint](http://eslint.org/) provides all the style checks JSHint offers, while being more configurable, more modular, and more extensible. I believe it is slightly slower, though I've never  experienced serious performance issues.

I'd suggest recommending it rather than JSHint. 

I'd at least tone down the claim that JSHint is the best to "JSHint is a good style checker".

**Comment by allanesquina** [comment]
I agree, to solve this, we can make a list of the lints|hints available with your specifications. What do you think?

**Comment by NateEag** [comment]
sorry, been extremely busy for the past month.

Just submitted a possible edit - let me know what you think.

**Comment by darlanmendonca** [comment]
I agree with eslint as best tool for lint:

- have the same rules like jshint
- but is flexible, to add plugins

an example, in specific frameworks, we have specific patterns to their., example, eslint-plugin-angular add rules to lint pattern of john papa

**Comment by darlanmendonca** [comment]
I vote to change in side jsHint for eslint

**Comment by darlanmendonca** [comment]
or metion both

**Comment by alex-shamshurin** [comment]
I think it's tern-lint

**Comment by devdesiignn** [comment]
ESLint is the best, works so well for beginners
